paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButton.kt

```diff
@@ -133,7 +154,10 @@ internal class PrimaryButton @JvmOverloads constructor(
                 originalLabel = text
             }
             viewBinding.label.setContent {
-                LabelUI(label = text)
+                LabelUI(
+                    label = text,
+                    color = defaultLabelColor,
+                )
```

Comment:
I’m not entirely clear on why we need this now?

Reply:
We need to be able to change the label color when user switches between light/dark theme.